daemon/src/maker_cfd.rs

```diff
@@ -271,8 +271,13 @@ where
     ) -> Result<()> {
         tracing::info!(%order_id, %taker_id,  "Received rollover proposal from taker");
 
+        let mut conn = self.db.acquire().await?;
+        let cfd = load_cfd(order_id, &mut conn).await?;
+        let position = cfd.position();
```

Comment:
this would conflict with: #1585 - suggest syncing with @thomaseizinger

Reply:
Yes, lets not do this please.